metagpt/config.py

```diff
@@ -125,6 +125,7 @@ def get_default_llm_provider_enum(self) -> LLMProviderEnum:
         if provider:
             logger.info(f"API: {provider}")
             return provider
+        return LLMProviderEnum.OPENAI
```

Comment:
no need add this, previous code has add DEFAULT_PROVIDER.
but it seems missing DEFAULT_PROVIDER here ==